pep/app/pods/components/search/item/bibliographic/template.hbs

```diff
@@ -32,7 +32,9 @@
             @icon="heart"
             @type="link"
             @tooltip={{t (if this.favorited "search.item.removeFromFavorites" "search.item.addToFavorites")}}
+            @tooltipVisible={{or this.media.isDesktop this.media.isJumbo}}
             class="p-0"
+            title={{t (if this.favorited "search.item.removeFromFavorites" "search.item.addToFavorites")}}
```

Comment:
when the tooltip is active on desktop, does it automatically remove this `title` for you, so both dont show if you hover long enough?